src/NotificationService/appsettings.json

```diff
@@ -1,2 +1,5 @@
 ﻿{
-}
+  "Slack": {
+    "WebhookUrl": "https://hooks.slack.com/services/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"
```

Comment:
Moeten we hier iets met secrets doen?

Reply:
ja ik weet zoiezo nog niet zo goed wat handig is hiermee. want ieder kanaal / persoon moet een eigen webhook krijgen.